### PR TITLE
URL Scraper using Node JS

### PR DESCRIPTION
**Pull request by Anushkabh**
# Pull Request for WebMasterLog 💡

## Issue Title: URL Scraper using Node JS

- **Name of the project** :  URL-Scraper
- **Category:** NodeJS
- **Name:** Anushka
- **GitHub ID:** Anushkabh
- **Idenitfy yourself:** JWOC contributor

Closes: #49 

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made

## Type of change ☑️

What sort of change have you made:

- [] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested? ⚙️

Describe how it has been tested postman
Describe how have you verified the changes made

## Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by Anushkabh** [comment]
@Avdhesh-Varshney  kindly review the pr

File: Node-JS-Projects/Intermediate/URL-Scrapper/README.md
Comment (Avdhesh-Varshney) [inline Node-JS-Projects/Intermediate/URL-Scrapper/README.md:1]: Follow the repo README template

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
- Follow the contributing guidelines carefully.
- Include a working video and from how to start the application upto interaction.

**Comment by Avdhesh-Varshney** [comment]
@Anushkabh resolve these changes and you should properly linked your issue with pr

**Comment by Avdhesh-Varshney** [comment]
@Anushkabh this pr is commited in the main branch which is totally a wrong way. You should read the contributing guidelines carefully and again create the pull request OK.

**Comment by Anushkabh** [comment]
@Avdhesh-Varshney apologies for this..

**Pull request closed without merging**